app/controllers/userInputCheckboxes.js

```diff
@@ -0,0 +1,12 @@
+// async
```

Comment:
I think you don't need this file since you're just posting the filter to "/users" or "/filter", so you can get the input values from `req.body` etc

Reply:
I first thought that i had to make separate files for making all the pieces of code that i'd have to use, but i quickly realised i didn't, so i already deleted them.